projects/admin/src/app/record/detail-view/document-detail-view/holding/holding.component.ts

```diff
@@ -68,59 +64,81 @@ export class HoldingComponent implements OnInit, OnDestroy {
     private _translateService: TranslateService,
   ) { }
 
-  /** Init */
+  /** onInit hook */
   ngOnInit() {
     this.holdingType = this.holding.metadata.holdings_type;
     if (this.holdingType !== 'electronic') {
       this._loadItems();
     }
-    this.getPermissions();
+    this._getPermissions();
   }
 
-  /** Destroy */
+  /** onDestroy hook */
   ngOnDestroy() {
     if (this.itemsRef != null) {
       this.itemsRef.unsubscribe();
     }
   }
 
-  /**
-   * Reload items when they are displayed.
-   */
+  /** Get permissions */
+  private _getPermissions(): void {
+    this._recordPermissionService
+      .getPermission('holdings', this.holding.metadata.pid)
+      .subscribe(permissions => this.permissions = permissions);
+  }
+
+  /** Load the items corresponding to a given holding PID. */
+  private _loadItems(): void {
+    let query = `holding.pid:${this.holding.metadata.pid}`;
+    let sort = '';
+    if (this.holding.metadata.holdings_type === 'serial') {
+      query += ' AND -issue.status:(claimed OR deleted OR late)';
+      sort = '-issue_expected_date';
+    }
+    this.itemsRef = this._recordService
+      .getRecords('items', query, 1, RecordService.MAX_REST_RESULTS_SIZE, [], {}, null, sort)
+      .subscribe((result: Record) => {
+        this.items = (this._recordService.totalHits(result.hits.total) > 0) ? result.hits.hits : null;
+        this.totalItemsCounter = this._recordService.totalHits(result.hits.total);
+      });
+  }
+
+  // COMPONENT FUNCTIONS ======================================================
+  /** Reload items when they are displayed. */
   toggleCollapse() {
     if (this.isItemsCollapsed) {
       this._loadItems();
     }
     this.isItemsCollapsed = !this.isItemsCollapsed;
   }
 
-  /** Remove an item given the item PID.
-   *
-   * @param itemPid - the PID of the item to remove.
+  /**
+   * Remove an item given the item PID.
+   * @param itemPid: string - the PID of the item to remove.
    */
-  deleteItem(itemPid) {
-    this._recordUiService.deleteRecord('items', itemPid).subscribe((success: any) => {
-      if (success) {
-        this.items = this.items.filter((i: any) => itemPid !== i.metadata.pid);
-        if (this.items.length === 0) {
-          this.deleteHolding.emit({holding: this.holding, callBackend: false});
+  deleteItem(itemPid: string): void {
+    this._recordUiService.deleteRecord('items', itemPid).subscribe(
+      (success: boolean) => {
+        if (success) {
+          this.items = this.items.filter((i: any) => itemPid !== i.metadata.pid);
+          if (this.items.length === 0) {
+            this.delete(true);
```

Comment:
Can you add a comment to specify that the backend remove the holding when the last item is removed.